### ISSUE TITLE
NameFormat errors aren't detected in function arguments

### ISSUE THREAD

**Issue by vonkanehoffen** [brigade/scss-lint#663]
Take this statement:
```
.whatever {
  background: $badVariableName;
}
```
It generates a `NameFormat` linting error as it should, because the variable name is CamelCase, not `hyphenated_lowercase` as it should be (see default [here](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#nameformat)).

Now take this one:
```
.whatever {
  background: lighten($badVariableName, 10%);
}
```
No error. There should be, I believe! :-)

Seen in v0.43.2

**Comment by sds** [comment]
Thanks for the report, @vonkanehoffen!

Fixed in f8a92f9c3788d976c8cba79bacc92f48179aeb46.
